libs/wallet/src/web3modal/hooks/useAccountsLoader.ts

```diff
@@ -0,0 +1,29 @@
+import { useWalletInfo as useWeb3WalletInfo } from '@web3modal/ethers5/react'
+import { useMemo } from 'react'
+import { TREZOR_CONNECTOR_ID, trezorProvider } from '../connectors/trezor/trezorConnector'
+
+const accountsLoaders: Record<string, WalletAccountsLoader> = {
```

Comment:
Nitpick: this is trezor specific, maybe it should be renamed and/or moved closer to the trezor connector